libraries/AP_Mount/AP_Mount.cpp

```diff
@@ -392,8 +392,22 @@ const AP_Param::GroupInfo AP_Mount::var_info[] = {
     // @Values: 0:None, 1:Servo, 2:3DR Solo, 3:Alexmos Serial, 4:SToRM32 MAVLink, 5:SToRM32 Serial
     // @User: Standard
     AP_GROUPINFO("2_TYPE",           42, AP_Mount, state[1]._type, 0),
+
+    // @Param: 2_RCINVLD_MODE
+    // @DisplayName: Mount2 RC Invalid Point Position
+    // @Description: Uses the designated mode's target position if RC invalid in RC Targeting mode
+    // @Values: -1:No Change,0:Retracted,1:Neutral
+    // @User: Standard
+    AP_GROUPINFO("2_RCINVLD_MODE", 43, AP_Mount, state[1]._rcinvalid_mode, -1),
 #endif // AP_MOUNT_MAX_INSTANCES > 1
 
+    // @Param: _RCINVLD_MODE
+    // @DisplayName: Mount RC Invalid Point Position
+    // @Description: Uses the designated modes target position if RC invalid in RC Targeting mode
+    // @Values: -1:No Change,0:Retracted,1:Neutral
+    // @User: Standard
+    AP_GROUPINFO("_RCINVLD_MODE", 44, AP_Mount, state[0]._rcinvalid_mode, -1),
```

Comment:
maybe let's name this parameter MNT_FS_RC_ACTION

Reply:
This was done.